### ISSUE TITLE
DibiPdoDriver

### ISSUE THREAD

**Issue by bckp** [dg/dibi#170]
Looks like dblib do not support PDO::getAttribute(PDO::ATTR_SERVER_VERSION)

Possible solution is
```php
$this->connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);

try {
    $this->serverVersion = $this->connection->getAttribute(PDO::ATTR_SERVER_VERSION); 	
} catch (PDOException $e) {
    $this->serverVersion = (isset($config['version']) ? $config['version'] : null);
}
```
But we force error mode to exception (that should be set anyway, becouse you expect exception in connect part) and add new variable to the config, as fallback if driver do not support this, but we want to specify version.

Another solution
```php
$this->serverVersion = @$this->connection->getAttribute(PDO::ATTR_SERVER_VERSION);	// intentionally @

if(!$this->serverVersion)
    $this->serverVersion = isset($config['version']) ? $config['version'] : null;
```
This is easier to add, do not change mode

Another solution is to add getAttribute method to the PdoDriver, but im not sure with this either.

**Comment by bckp** [comment]
@dg What is your opinion?

**Comment by bckp** [comment]
Fixed in #171
